src/style.css

```
@@ -1 +1,70 @@
 /* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  text-align: center;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.header__logo {
+  padding: 10px 0;
+  width: 40px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item:not(:last-child) {
+  padding-right: 20px;
+}
+
+.nav__link {
+  position: relative;
+  text-transform: uppercase;
+  display: block;
+  text-decoration: none;
+  color: #000;
+  padding: 23px 0;
+}
```

Let's add fixed height 60px and center them vertically, instead of paddings
```suggestion
.nav__link {
  position: relative;
  text-transform: uppercase;
  display: block;
  text-decoration: none;
  color: #000;
}
```